admin/sections/config/providers/openai.tsx

```diff
@@ -44,10 +46,45 @@ export const OpenAIForm = ({
           </FormItem>
         )}
       />
-      <ModelForm
-        form={form}
-        defaultValues={defaultValues}
-        supportedModels={supportingModels}
+      <div className="flex space-x-4">
+        <div className="w-1/2">
+          <ModelForm
+            form={form}
+            defaultValues={defaultValues}
```

Comment:
isn't it easier to just pass one value?
```suggestion
            defaultValue={defaultValues.model}
```